docs/Deep-Dives/metaapi.md

```diff
@@ -0,0 +1,200 @@
+
+
+# Flotiq + Meta-API + Google Spreadsheets use case
+
+This guide shows how to add your private Flotiq API to Meta-API and prepare an example Spell. 
+In our case we will create a report in Google Spreadsheet based on products stock quantity.
+
+![Integrate Flotiq with Google Spreadsheets using Meta-API Spell](images/metaapi/img-0.png){: .border}
+
+
+## 1. Create an empty sheet in Google Spreadsheets
+
+Let's prepare our spreadsheet to use in further steps. 
+We add two columns (Date, Stock Quantity) and a chart with the default config for these columns.
+
+![Create an empty sheet to store Flotiq aggregated data](images/metaapi/meta-spr-1.png){: .border}
+
+Notice the `spreadsheet id`. 
+It's located in the URL `https://docs.google.com/spreadsheets/d/[___here___]/edit#gid=0`. 
+We will use it in the next steps.
+
+
+## 2. Model your data in Flotiq
+
+Sign in to your Flotiq account and make sure that you have added your Content Type Definitions. 
+In our example, we use [predefined Product CTD](https://flotiq.com/docs/panel/predefined-content-types/) with an extra property named `stock_quantity`.
+
+!!! Note
+    If you are new to Flotiq, check the [Dashboard > Content Types](https://flotiq.com/docs/panel/content-types/)
+    section in the docs
+
+![Product Content Type Definition in Flotiq](images/metaapi/meta-flotiq-1.png){: .border}
+
+Now it's time to add some example data. 
+As we are creative, we added more sophisticated data than "Lorem ipsum" :)
+
+![List of Content Objects in Flotiq](images/metaapi/meta-flotiq-2.png){: .border}
+
+
+### 3. Get your OpenAPI Specification
+
+This is a crucial step in Flotiq. 
+Let's open your `API Doc` and download `OpenAPI Specification`.
+
+![Link to your personal API Doc](images/metaapi/meta-flotiq-3.png){: .border}
+
+![Download OpenAPI Specification button](images/metaapi/meta-flotiq-4.png){: .border}
+
+We use the content of the downloaded file in the next step.
+
+
+### 4. Add your Flotiq API to Meta-API
+
+!!! Note "Why there is no predefined Flotiq API in Meta-API catalog?"
+    The Main concept of Flotiq is that your API Specification literally represents your set of content definitions.
+    Your API represents your endpoints, with your models consisting of your properties.
+    In short, **Flotiq API contains different endpoints for each user to be more personalized**.
+
+We assume that you have an account on Meta-API platform and you are logged in. 
+Go to the Catalog of API and click "Add an API".
+
+![Add Flotiq API to Meta-API catalog](images/metaapi/meta-1.png){: .border}
+
+In the `Add an API to the catalog` modal, go to the bottom and fill the `Open API Specification` field with data
+retrieved in step 2. Other fields (except Category) should auto-complete.
+
+![Add an API dialog is the place where you paste OpenAPI Specification](images/metaapi/meta-2.png){: .border}
+
+And that's it. We have our private Flotiq API in our private Meta-API Catalog. 
```

Comment:
maybe something like, 
Please note that every time your flotiq model changes ( and your API with it) you might need to update your meta API connection with new open-api specification